File: Actividad 5/Problema 3/problema3_cs/FechaTest.cs
using System;
class FechaTest
{
    public static void Main(string[] args)
        {
            //FechaTest
            //Crear primer objeto
            Fecha fecha1 = new Fecha();
                
            Console.WriteLine("Ingresa el dia: ");
            fecha1.dia = int.Parse(Console.ReadLine());
            
            Console.WriteLine("Ingresa el mes (numero): ");
            fecha1.mes = int.Parse(Console.ReadLine());
                
            Console.WriteLine("Ingresa el año: ");
            fecha1.anio = int.Parse(Console.ReadLine());
                    
            fecha1.mostrarFecha();
    
         }
}
    

File: Actividad 5/Problema 3/problema3_java/FechaTest.java
import java.util.Scanner;
public class FechaTest
{
        public static void main(String args[])
        {
            //FechaTest
            //Crear primer objeto
            Fecha fecha1 = new Fecha();
            Scanner input = new Scanner(System.in);

        
            System.out.println("Ingresa el dia: ");
            fecha1.dia = input.nextInt();

            System.out.println("Ingresa el mes (numero): ");
            fecha1.mes = input.nextInt();
        
            System.out.println("Ingresa el año: ");
            fecha1.anio = input.nextInt();
            
            fecha1.mostrarFecha();
        }
}
    